### ISSUE TITLE
[PL] UGC feature to be added.

### ISSUE THREAD

**Issue by saurabhtz** [vinodtapmobi/havok-operations#75]
Need to add UGC(User Generated Content) feature.

**Comment by robintapmobi** [comment]
- Team we need to add option where end user can upload his own content { Required when client want to play contest- Win cutest baby contest }.
- User can only upload the content, publishing or taking it live will be done only after Moderation could be done by client or support team.
- User will be informed regarding Moderation (uploaded content will be live in 24 Hrs or 48 Hrs or after client approves)

**Comment by vinodtapmobi** [comment]
On Hold.
